src/components/annotate/BoundingBox.vue

```diff
@@ -65,6 +65,9 @@
                             :show-labels="false"
                             :allow-empty="false"
                             :searchable="false"
+                            :style="DEVstyle"
+                            :open-direction="openDirection"
```

Comment:
Used the component API to modify its  shape depending on the bounding box position